command-line/test/Command/ScriptSpec.hs

```diff
@@ -55,27 +58,85 @@ spec = do
         specScriptInvalid Malformed
             [iii|any [ #{verKeyH1}, ]|]
 
-        specScriptInvalid ListTooSmall
+        specScriptHashProper "script10w5mpzsfwcjeg4ksqh7ffl0ya4fda0yyra7z7c3je6g67x57ctn"
             [iii|at_least 4 [ #{verKeyH1}, #{verKeyH2}, #{verKeyH3} ]|]
 
-        specScriptInvalid ListTooSmall
+        specScriptHashProper "script16app2n3qlwc543cmvrlnxt29nrpqttu2kc72ftj5fapvzccsc6y"
             [iii|at_least 1 [ #{verKeyH1}, at_least 2 [ #{verKeyH2} ] ]|]
 
-        specScriptInvalid EmptyList
+        specScriptHashProper "script163qjya2n5rc6je07ultq5rmjfvmgm5dam0pqsuc0en4u7967saj"
             [iii|all []|]
 
-        specScriptInvalid EmptyList
+        specScriptHashProper "script1duhpq0x5k2tey5rltut0ucunhqagxfhwqyu5rh4jmrw2qrjtkus"
             [iii|any [ #{verKeyH1}, all [] ]|]
 
-        specScriptInvalid MZero
+        specScriptHashProper "script1njfxl9tzwfqaxk8tt5llh9zdfse9srvpeunjssjslrhvvl0ys30"
             [iii|at_least 0 [ #{verKeyH1}, #{verKeyH2} ]|]
 
-        specScriptInvalid DuplicateSignatures
+        specScriptHashProper "script12zfnrxhgn9xmzpptgz2eaepf8w9qww460xm0ek453y8qx3kfk7p"
+            [iii|at_least 1 [ #{verKeyH1}, #{verKeyH2}, active_from 10, active_until 25 ]|]
+
+        specScriptHashProper "script1uan07g0z30cfpvg7gxp5fmvjmv6ylaqa84msw8twyt5777hf7gf"
             [iii|any [ #{verKeyH1}, #{verKeyH2}, #{verKeyH1}]|]
 
         specScriptInvalid Malformed
             [iii|script_vkh18srsxr3khll7vl3w9mqfu55n6wzxxlxjq8egs9|]
 
+        specScriptInvalid Malformed
+            [iii|any [ #{verKeyH1}, #{verKeyH2}, active_from a]|]
+
+    describeCmd [ "script", "validate"] $ do
```

Comment:
What about splitting this module in two: `test/Command/Script/HashSpec` and `test/Command/Script/ValidateSpec` to follow the same pattern as other test files.

Reply:
done, see HEAD